README.md

```diff
@@ -1,19 +1,26 @@
-# LLMNETES
+# LLM*NETES
 
-A Kubernetes controller that allows users to operate their kubernetes using natural language.
+Bringing LLM's to Kubernetes.
 
-## Description
+<p align="center">
+<img src="./docs/images/llmnetes-cute-astronaut.jpg" width="250" >
+</p>
+
+Disclaimer: This is a project is a work in progress and is not ready for
```

Comment:
typo: > This is a project is a work in progress

This project is work in progress